src/leibniz.rs

```diff
@@ -8,18 +8,14 @@ fn main() {
     file.read_to_string(&mut contents)
         .expect("something went wrong reading the file");
 
-    let rounds = &contents.trim().parse::<i32>().unwrap();
+    let rounds = &contents.trim().parse::<u32>().unwrap() + 2;
 
     // We need to use f64 instead of f32 because otherwise the precision would be off
-    let mut x: f64 = 1.0;
-    let mut pi: f64 = 1.0;
-
-    for i in 2..(rounds + 2) {
+    let mut x: f32 = 1.0;
```

Comment:
With the changes in #24 (or another PR that does similar) there's no need to sacrifice precision for a few nanoseconds.
```suggestion
    let mut x = 1.0;
```

Reply:
@workingjubilee so you suggest not to define the type to keep precision as far as I understand?
@gauteh what are your thoughts. Since I merged #24 this could be a valid change.

Reply:
Yep, then this change should also be included. By not defining the type, rust automatically infers f64 in this case since it can then be added to `pi` without conversion later. So it could also explicitly be changed to:

```
let mut x: f64 = 1.0;
```